### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by prajapati-priyanka**
-add button component.
-not responsive, will make it in the end.

csb link: https://codesandbox.io/s/github/prajapati-priyanka/pineUI-component-library/tree/button-component

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]

File: component/button/button.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                        </div>
+                        <iframe src="https://www.thiscodeworks.com/embed/62027870234b8a0015700b9b" style="width: 100%; height: 190px;" frameborder="0"></iframe>
+                    </div>
+                </article>
+                <article class="comp-desc">
+                    <h2 class="comp-heading">Floating Button</h2>
+
+                    <p class="comp-heading-desc lg-text">
+                        Floating-action button is usually fixed to right-bottom corner.
+
```
Comment (vermasavita) [inline component/button/button.html]: Remove this extra space. Code will look clean

File: component/button/button.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+                                <span class="floating-btn x-lg-text">
+                                    <i class="fa fa-plus"></i>
+                                </span>
+                            </div>
+                            <div class="floating-btn-box">
+                                <span class="floating-btn x-lg-text">
+                                    <i class="fa fa-trash"></i>
+                                </span>
+                            </div>
+                    </div>
```
Comment (vermasavita) [inline component/button/button.html]: Otherwise looks good to me. 

Great work!!

File: component/button/button.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                        <span class="highlight">btn-primary</span>,
+                        <span class="highlight">btn-secondary</span>and
+                        <span class="highlight">btn-danger</span>
+
+                    </p>
+                    <div class="component-code lg-text">
+                        <div class="container-flex">
+
+                            <button class="btn btn-primary">Primary</button>
+                            <button class="btn btn-secondary">Secondary</button>
```
Comment (vermasavita) [inline component/button/button.html]: You can choose a different color for secondary. It's kind of same as danger color.

**Review by vermasavita — APPROVED** [review]

File: component/button/button.html (hunk 1 above)
Comment (prajapati-priyanka) [reply component/button/button.html]: Okay, I will do that.

File: component/button/button.html (hunk 2 above)
Comment (prajapati-priyanka) [reply component/button/button.html]: Okay. Thanks

File: component/button/button.html (hunk 3 above)
Comment (prajapati-priyanka) [reply component/button/button.html]: I will do that. Thanks.

**Pull request merged**